src/dynamic_boundary_conditions/rec1_osm_match.py

```diff
@@ -15,7 +15,7 @@
 from shapely.geometry import Point
 from sqlalchemy.engine import Engine
 
-from src.lidar import dem_metadata_in_db
+from newzealidar.utils import get_dem_band_and_resolution_by_geometry
```

Comment:
```suggestion
import sys
sys.path.insert(0, r'../NewZeaLiDAR')
from newzealidar.utils import get_dem_band_and_resolution_by_geometry
```
Needs this to be able to run